src/ocnicepost.fd/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+list(APPEND fortran_src
+arrays_mod.F90
+init_mod.F90
+masking_mod.F90
+ocnicepost.F90
+utils_mod.F90
```

Comment:
Not well-versed in cmake. Can this be indented?

Reply:
done.